makederiv

```diff
@@ -18,8 +18,9 @@ _usage(){
         echo "      youtube (mp4) for uploading"
         echo "      resourcespace (mp4) for uploading to resourcespace"
         echo "      dvd (iso)"
+        echo "      m4a (m4a)"
```

Comment:
I suggest we remove m4a as an option and instead just have a conditional podcast option which produces an m4a if the input is audio-only.

Reply:
removed